<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStudentWidget</class>
 <widget class="QWidget" name="AddStudentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1639</width>
    <height>1129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="header">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>50</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Добавление студента</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>210</y>
     <width>421</width>
     <height>891</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: #007BFF; /* Синий фон для окна */
    border-radius: 12px; /* Закругленные углы */
    padding: 20px; /* Отступы внутри окна */
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="surnameEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #FFFFFF; /* Белый фон для полей ввода */
    border: 2px solid #0056b3; /* Синяя рамка */
    border-radius: 6px; /* Закругленные углы */
    padding: 10px; /* Отступы внутри поля */
    font-size: 22px; /* Размер шрифта */
    margin: 5px 0; /* Отступы между полями */
}

QLineEdit:focus {
    border-color: #007BFF; /* Цвет рамки при фокусе */
}</string>
    </property>
    <property name="placeholderText">
     <string>Фамилия</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #FFFFFF; /* Белый фон для полей ввода */
    border: 2px solid #0056b3; /* Синяя рамка */
    border-radius: 6px; /* Закругленные углы */
    padding: 10px; /* Отступы внутри поля */
    font-size: 22px; /* Размер шрифта */
    margin: 5px 0; /* Отступы между полями */
}

QLineEdit:focus {
    border-color: #007BFF; /* Цвет рамки при фокусе */
}</string>
    </property>
    <property name="placeholderText">
     <string>Имя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="patronymEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #FFFFFF; /* Белый фон для полей ввода */
    border: 2px solid #0056b3; /* Синяя рамка */
    border-radius: 6px; /* Закругленные углы */
    padding: 10px; /* Отступы внутри поля */
    font-size: 22px; /* Размер шрифта */
    margin: 5px 0; /* Отступы между полями */
}

QLineEdit:focus {
    border-color: #007BFF; /* Цвет рамки при фокусе */
}</string>
    </property>
    <property name="placeholderText">
     <string>Отчество</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>800</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #0056b3; /* Основной цвет фона кнопки */
    color: white; /* Цвет текста кнопки */
    border: none; /* Убираем рамки */
    border-radius: 8px; /* Закругленные углы */
    padding: 10px; /* Отступы внутри кнопки */
    font-size: 22px; /* Размер шрифта кнопки */
    font-weight: bold; /* Жирный текст */
    margin-top: 20px; /* Отступ сверху */
}

QPushButton:hover {
    background-color: #004085; /* Темнее при наведении */
}

QPushButton:pressed {
    background-color: #003366; /* Цвет при нажатии */
}</string>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="phoneNumberEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #FFFFFF; /* Белый фон для полей ввода */
    border: 2px solid #0056b3; /* Синяя рамка */
    border-radius: 6px; /* Закругленные углы */
    padding: 10px; /* Отступы внутри поля */
    font-size: 22px; /* Размер шрифта */
    margin: 5px 0; /* Отступы между полями */
}

QLineEdit:focus {
    border-color: #007BFF; /* Цвет рамки при фокусе */
}</string>
    </property>
    <property name="placeholderText">
     <string>Номер телефона</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ageEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #FFFFFF; /* Белый фон для полей ввода */
    border: 2px solid #0056b3; /* Синяя рамка */
    border-radius: 6px; /* Закругленные углы */
    padding: 10px; /* Отступы внутри поля */
    font-size: 22px; /* Размер шрифта */
    margin: 5px 0; /* Отступы между полями */
}

QLineEdit:focus {
    border-color: #007BFF; /* Цвет рамки при фокусе */
}</string>
    </property>
    <property name="placeholderText">
     <string>Возраст</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="blockEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #FFFFFF; /* Белый фон для полей ввода */
    border: 2px solid #0056b3; /* Синяя рамка */
    border-radius: 6px; /* Закругленные углы */
    padding: 10px; /* Отступы внутри поля */
    font-size: 22px; /* Размер шрифта */
    margin: 5px 0; /* Отступы между полями */
}

QLineEdit:focus {
    border-color: #007BFF; /* Цвет рамки при фокусе */
}</string>
    </property>
    <property name="placeholderText">
     <string>Номер блока</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="optEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>730</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #FFFFFF; /* Белый фон для полей ввода */
    border: 2px solid #0056b3; /* Синяя рамка */
    border-radius: 6px; /* Закругленные углы */
    padding: 10px; /* Отступы внутри поля */
    font-size: 22px; /* Размер шрифта */
    margin: 5px 0; /* Отступы между полями */
}

QLineEdit:focus {
    border-color: #007BFF; /* Цвет рамки при фокусе */
}</string>
    </property>
    <property name="placeholderText">
     <string>Отработанные часы ОПТ</string>
    </property>
   </widget>
   <widget class="QComboBox" name="activityType">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semilight</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    background-color: #FFFFFF; /* Белый фон для ComboBox */
    border: 2px solid #0056b3; /* Синяя рамка */
    border-radius: 6px; /* Закругленные углы */
    padding: 10px; /* Отступы внутри ComboBox */
    font-size: 22px; /* Размер шрифта */
    color: #000000; /* Цвет текста */
    padding-right: 30px; /* Отступ справа для стрелки */
}

QComboBox::down-arrow {
    /*image: url(&quot;C:\\Users\\Сергей\\source\\repos\\CloneDormitory\\CloneDormitory\\CloneDormitory\\down-arrow.png&quot;);  Замените на ваш путь к изображению стрелки */
    width: 10px; /* Ширина стрелки */
    height: 10px; /* Высота стрелки */
}

QComboBox::item {
    background-color: #FFFFFF; /* Фон для элементов выпадающего списка */
    color: #000000; /* Цвет текста элементов */
}

QComboBox::item:selected {
    background-color: #007BFF; /* Фон для выбранного элемента */
    color: #FFFFFF; /* Цвет текста для выбранного элемента */
}</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
